qiskit/transpiler/target.py

```diff
@@ -287,7 +290,9 @@ def __init__(
                 matches the qubit number the properties are defined for. If some
                 qubits don't have properties available you can set that entry to
                 ``None``
-        Raises:
+            meas_map(list): A list of sets of qubits that must be
+                measured together. The qubits which should be measured concurrently
+                is provided as the nested list like [[0, 1], [2, 3, 4]].
```

Comment:
Can we come up with a more descriptive name than `meas_map`. We're not tied to the name in the old configuration payload in the target and I think `meas_map` doesn't really mean anything to people. It's important for the target that we're building a data model that's descriptive for all backends, not just IBM's current model.

Maybe something like `concurrent_measurements` is more descriptive?

Reply:
concurrent_measurements also looks good to me.

Reply:
Thanks. I updated at https://github.com/Qiskit/qiskit-terra/pull/10258/commits/e49b31951c647ac6aec909a59c0977f8a913f059 .